js-exercises/accessor-properties/accessorProperties.js

```diff
@@ -0,0 +1,10 @@
+const accessorProperties = () => ({
+  binary: null,
```

Comment:
As we don't want users to access the `binary` property directly, use the name `_binary` for it. 
As per the convention, variables with `_` prefix is internal variables and are not supposed to be used directly.